lib/pbench/agent/tool_meister.py

```diff
@@ -481,21 +545,24 @@ def start(self, env=None):
             self.logger.info("Started persistent tool %s, %r", self.name, self.args)
 
     def stop(self):
-        """stop - terminate the persistent tool sub-process.
+        """Terminate the persistent tool sub-process.
 
         This method does not wait for the process to actually exit. The caller
         should issue a wait() for that.
         """
         if self.process is None:
-            self.logger.error("Nothing to terminate")
+            self.logger.error("No process to stop")
             return
 
         # First try to gracefully terminate.
```

Comment:
"First"?  This is the only thing that this function does...we can probably do without this comment.